cfgv.py

```diff
@@ -162,6 +163,19 @@ def _no_additional_keys_check(self, dct):
         )
 
 
+def _warn_additional_keys_check(self, dct):
+    extra = sorted(set(dct) - set(self.keys))
+    logger = logging.getLogger(__name__)
```

Comment:
this will use `cfgv`'s logger which isn't necessarily right -- let's just keep this class in `pre_commit` for now

Reply:
Sorry, but not really understand. 
You mean that the logger can came from `pre_commit`? So we initialize logger in `pre_commit` and when WarnAdditionalKeys is called we can pass the logger there? And here `pre_commit`'s logger will be used.
Please correct me if I am wrong. Thanks.

Reply:
`__name__` here will be `cfgv` and so this will be `cfgv`'s logger and not pre-commit's

Reply:
I have updated my code (removed the internal `logger` dependency), but I am still not satisfied with my testcase.. 
Could you help me please in this; the code behind the cfgv.Map.check() is too complex for me (do not really understand the concept).
Thanks

Reply:
In my opinion it is not true that the cfgv.py:167 line is not run via the testcase (after checking the last CI failed). Why is it reported so?

Reply:
Sorry, still checking. It turned out the `extra` is `False` this is the main problem here.

Reply:
Oh :) Finally found it, with using `pytest ... --log-level=debug` in coverage we have 100% coverage.
Huhh it takes some time to find the root cause. (I have got new XP's :)).
So the question is can we add --log-level to tox.ini at coverage run?